### PR TITLE
Feature/num 391 integration tests invalid profil heart rate

### PR DESCRIPTION
**Pull request by af-dpe**
- Added heart rate example robot json
- Added Testcase 013 - 025
- Added Keyword _create Observation Heart Rate JSON_ (Observation.robot)
- Added new robot file: _json-manipulation.robot_
- added resource to suite settings
-

### CODE REVIEW

File: tests/robot/_resources/keywords/observation.robot [hunk 1]
```diff
@@ -226,3 +226,28 @@ create frailty scale score
...
+                        prepare new request session  Prefer=return=representation
+
+    &{resp}             Run Keywords
+                        ...    ehr.create new ehr               000_ehr_status.json                             AND
+                        ...    load JSON                        observation-example-heart-rate-robot.json       AND
+                        ...    update Resource Type             ${resourceType}                                 AND
+                        ...    update ID                        ${ID}                                           AND
+                        ...    update Meta Profile              ${meta}                                         ${profile}                     AND
+                        ...    update Status                    ${status}                                       AND
+                        ...    update Identifier                ${Identifieravailable}                          ${Identifiercodingsystem}      ${Identifiercodingcode}     ${Identifiersystem}     ${Identifiervalue}      AND
```
Comment (af-pwo) [inline tests/robot/_resources/keywords/observation.robot:244]: Parameter ${identifierAssignerReference} should be included here.

File: tests/robot/_resources/keywords/observation.robot [hunk 2]
```diff
@@ -226,3 +226,28 @@ create frailty scale score
...
+                        ...    update Category                  ${categoryavailable}                            ${categorycodingavailable}     ${categorysystem}           ${categorycode}         AND
+                        ...    update Code                      ${codeavailable}                                ${codecodingavailable}         ${code0system}              ${code0code}            ${code1system}          ${code1code}    AND
```
Comment (af-pwo) [inline tests/robot/_resources/keywords/observation.robot:246]: Parameter ${codeText} should be included here.

File: tests/robot/OBSERVATION/01_create.robot [hunk 3]
```diff
@@ -141,3 +142,302 @@ Force Tags              create
...
+    Observation    	    	heart-rate  				true        http://hl7.org/fhir/StructureDefinition/heartrate        final			true      http://terminology.hl7.org/CodeSystem/v2-0203            Dave          https://www.charite.de/fhir/CodeSystem/observation-identifiers        8867-4_HeartRate       true             true         http://terminology.hl7.org/CodeSystem/observation-category         vital-signs        true			   true			  				http://loinc.org		  				8867-4				http://snomed.info/sct         		   abcd    		   true    		 	  valid      		2020-02-25		  true		   ${-10}	     pro Minute	       http://unitsofmeasure.org    	   /min			 false               422           0    	 There was a problem saving the compositionThere was a Error in saveHeartRate                                                                         There was a problem saving the compositionThere was a Error in saveHeartRate
```
Comment (af-pwo) [inline tests/robot/OBSERVATION/01_create.robot:358]: These two cases are failing in a local test run. They dont result in an 422, but in a 201. The documentation also has no constraints for decimal numbers for this parameter. Should we comment these out/move these to the valid test cases?

**Review by af-pwo — COMMENTED** [review]
src/test/resources/Observation/observation-example-heart-rate-robot.json
 src/test/resources/Observation/observation-example-heart-rate.json

Why do we need both of these files?

File: tests/robot/OBSERVATION/01_create.robot (hunk 3 above)
Comment (af-dpe) [reply tests/robot/OBSERVATION/01_create.robot:358]: @af-pwo works for me like we discussed in slack, not sure why it fails for you, but its also working in the CI

File: tests/robot/_resources/keywords/observation.robot (hunk 2 above)
[the remark this answers is not in the document]
Comment (af-dpe) [reply tests/robot/_resources/keywords/observation.robot:246]: you have 3 types of text for code in this example, can add this but would like to do it in a new branch ;)

File: tests/robot/_resources/keywords/observation.robot (hunk 1 above)
[the remark this answers is not in the document]
Comment (af-dpe) [reply tests/robot/_resources/keywords/observation.robot:244]:
```
            "assigner": {
                "reference": "Organization/Charité"
```
we add this in an extra branch

**Comment by af-dpe** [comment]
> src/test/resources/Observation/observation-example-heart-rate-robot.json
> src/test/resources/Observation/observation-example-heart-rate.json
> 
> Why do we need both of these files?

First on is for the Robot testcase -> has dataabsentreason 
Second for the Java Tests and the shorter tests with robot -> has not dataabsentreason

also added the JSOn because it was not running in the java tests anymore after adding some values

**Pull request merged**